### DISCUSSION TITLE
Welcome to mikefm-skill Discussions!

### DISCUSSION THREAD

**Discussion by jsmariegaard** [DHI/modelskill#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hendrik1987** [comment]
Hi all,
Great initiative!
I'm Hendrik, Metocean engineer at DHI-POT. Like for many of us, MIKE FM model assessment and calibration is like bread and butter. Any tool which will help to streamline the workflow and automate tedious data handling tasks are very welcome. The less engineers have to do of the latter, the more we can focus on the important stuff, i.e., make less errors and get more out of our models. Even more welcome is the community approach, which will ensure cross-department cooperation. Looking forward to the developments and I will try my best to contribute.
Cheers, Hendrik
